moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/JAXBEnumTypeConverter.java

```diff
@@ -92,6 +95,16 @@ public void initialize(DatabaseMapping mapping, Session session) {
         super.initialize(mapping, session);
     }
 
+    private String getEnumValue(Enum theEnum) {
+        try {
+            Field field = theEnum.getClass().getField(theEnum.name());
```

Comment:
Please use (maybe add new method into) https://github.com/eclipse-ee4j/eclipselink/blob/master/moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/ReflectionUtils.java
to ensure, that `PrivilegedAccessHelper.callDoPrivilegedWithException(` is used instead of direct call.